kuma/wiki/helpers.py

```diff
@@ -324,8 +325,12 @@ def wiki_url(context, path):
     Create a URL pointing to Kuma.
     Look for a wiki page in the current locale, or default to given path
     """
+    request = context['request']
+    if waffle.flag_is_active(request, 'dumb_wiki_urls'):
```

Comment:
Yeah, I think we should make the "dumb" the default as soon as possible and live with the redirecting nature of the URLs